src/components/Cryptos/Cryptos.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import './Cryptos.scss';
+import NavbarCryptos from '../Navbar/NavbarCryptos';
+
+const Cryptos = () => {
+  const currenciesData = useSelector((store) => store.currencies);
+
+  return (
+    <>
+      <NavbarCryptos />
+      <div className="cryptos-container">
+        {
+          currenciesData.currencies.map((currency) => (
+            <Link className="crypto-link d-flex flex-column justify-content-center align-items-center bg-primary text-white m-2 rounded-4" to="/details" key={currency.id} state={{ state: currency }}>
+              <h3 className="text-center">{currency.name}</h3>
+              <span>{currency.symbol}</span>
+              <p>
+                {
+                Number(currency.price).toFixed(5)
+                }
+                $
+              </p>
+            </Link>
+          ))
+        }
+      </div>
```

Comment:
- Hey, I love your project, this is a great one :+1:  however, as it required that you stick to the design layout, we need to improve on this to make our design close to the given template design and to meet the project requirement which is key in having your project approved. Kindly see the image below to work on the improvement.

*template design*
![Screenshot from 2022-11-27 06-04-09](https://user-images.githubusercontent.com/59397159/204120229-e32c6acb-e17d-48eb-a920-7373a8aa9e2a.png)

*your design*
![Screenshot from 2022-11-27 06-00-50](https://user-images.githubusercontent.com/59397159/204120241-42818f41-f199-4112-b8dd-cb9b44c21881.png)

---

- As you can see the container has two columns and each content is in rows. Please, kindly improve the design layout and maintain one color scheme throughout the design.
- Align each content to match the design.